### ISSUE TITLE
[Cherry-Pick/Sharing] Support --force option in install script

### ISSUE THREAD

**Issue by tzellman** [mislav/dotfiles#11]
Thanks for providing your dotfiles publicly. I was able to make use of some of what you had, primarily the concept of installing a variety of managed dotfiles.

Since I've diverged quite a bit so far for my personal purposes, I can't really create an isolated pull request via github. Check out this commit if you are interested in this change:
https://github.com/tzellman/dotfiles/commit/53e97ed1ccc25ec07e50b599f1e6cd2ed25e1364

Basically, -f/--force forces an overwrite of existing files.

Cheers!

**Comment by mislav** [comment]
You could create an isolated pull request via GitHub quite easily:

1. Create a new branch in your repo based of `mislav:master`
2. Cherry-pick the commit in question to the new branch
3. Push said branch to your fork and submit a pull request.

That being said, I don't think I'll add this option to my `install` scripts simply because it would be too easy to shoot yourself in the foot with. I don't wish to give anyone a tool with which they can accidentally overwrite their own dotiles. So in order to inherit my dotfiles via symlinks, people need to ensure that their old dotfiles are moved away from their original location either by manually deleting them or backing them up in another location.
